testsuite/features/step_definitions/common_steps.rb

```diff
@@ -220,12 +220,12 @@
 end
 
 # action chains
-When(/^I check radio button "(.*?)"$/) do |arg1|
-  raise ScriptError, "#{arg1} can't be checked" unless choose(arg1)
-end
-
-When(/^I check radio button "(.*?)", if not checked$/) do |arg1|
-  choose(arg1) unless has_checked_field?(arg1)
+When(/^I check radio button "(.*?)"$/) do |radio_button|
```

Comment:
Would you like to remove step `When(/^I check radio button "(.*?)", if not checked)` ?

see https://github.com/uyuni-project/uyuni/blob/9c79063e60b6ff262277b005af250f6ddbc75a88/testsuite/features/step_definitions/common_steps.rb#L227

It seems to be now duplicating your refactored step

Reply:
It's already removed, if you notice I combined both in one. As I mentioned in the description.